packages/components/organisms/f-checkout/src/services/tests/mapper.test.js

```diff
@@ -0,0 +1,95 @@
+import mapCheckoutRequest from '../mapper';
+
+const defaultParams = {
+    address: {},
+    customer: {},
+    isCheckoutMethodDelivery: true,
+    time: {},
+    userNote: ''
+};
+
+describe('checkout mapper', () => {
+    it('should map customer details correctly', () => {
```

Comment:
Just out of curiosity, could this be an `it.each`?

Reply:
So I did try this to see how it worked out, but I don't think it works for two reasons:
1. I'm not a huge fan of putting the property names in magic strings
2. Note the property `mobileNumber` being mapped to `phoneNumber`, so the `it.each` needs another parameter which makes it more untidy.

```javascript
it.each([
    ['firstName', 'firstName', 'Test'],
    ['lastName', 'lastName', 'Tester'],
    ['mobileNumber', 'phoneNumber', '07890123456']
])('should map customer %s correctly', (property, expectedProperty, value) => {
    // Arrange
    const customer = { [property]: value };

    // Act
    const requestBody = mapCheckoutRequest({
        ...defaultParams,
        customer
    });

    // Assert
    expect(requestBody.customer[expectedProperty]).toBe(customer[property]);
});
```

What do you think @FayeCarter?

Reply:
Ahh I see, yeah sounds good to me